roles/network_plugin/multus/templates/multus-daemonset.yml.j2

```diff
@@ -22,6 +22,7 @@ spec:
         tier: node
         app: multus
     spec:
+      priorityClassName: system-node-critical
```

Comment:
HI  @Mosibi 

Comparing with https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/deployments/multus-daemonset.yml .

It's better to use a variable rather than changing the default setting. like
```
multus_priority_class_name
```

Another idea is to create a PR for the upstream(https://github.com/k8snetworkplumbingwg/multus-cni) first, then change the kubespray. :-）

Reply:
Hi @yankay,

My first thought was also to make it a variable, but looking at all other places in the Kubespray code where a priorityClassName is set, I found that it is set static at all places. Therefore to be consistent I did the same, but if you prefer to set it like your proposal, I will change my PR.

Reply:
The best practice is to keep the default setting with upstream. If there must be some customize, we can use variable :-)

Reply:
@yankay 

Okay, then I will submit a pull request at Multus and when that one is accepted and merged, open a one over here to update it. For now I will close this one.